wrapanapi/systems/msazure.py

```diff
@@ -766,6 +775,40 @@ def list_router(self):
     def disconnect(self):
         pass
 
+    def _list_resources(self, resource_group=None, hours_old=0):
+        """
+        Lists group resources older than hours_old
+        """
+
+        resources = []
+        now_time = datetime.utcnow().replace(tzinfo=pytz.utc)
+
+        for res in self.list_all_resources_by_resource_group(resource_group):
+            start_time = parser.parse(res.additional_properties['createdTime'])
+            timediff = now_time - start_time
+            totalhours = timediff.total_seconds() / 3600
+
+            if totalhours > hours_old:
```

Comment:
I'd be a bit more careful about the TypeError potential here, since `hours_old` is coming from the user.  A ValueError would be appropriate if `hours_old` can't be cast to a float for comparison.